src/test/java/gov/usds/case_issues/test_util/CaseListFixtureService.java

```diff
@@ -0,0 +1,156 @@
+package gov.usds.case_issues.test_util;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import gov.usds.case_issues.db.model.CaseManagementSystem;
+import gov.usds.case_issues.db.model.CaseType;
+import gov.usds.case_issues.db.model.TroubleCase;
+
+@Service
+@Profile("autotest")
+public class CaseListFixtureService {
```

Comment:
Not sure what this is for. I think I will need to see an example test first. I am confused by the 30 second recreation logic though.

Reply:
This is an extraction from the paging/filtering tests (see eb0fdf1de8a9f7e1d7c2355611c50dca6f95f34b), because having 36 test cases and all the setup infrastructure in there was unwieldy. 

The 30-second staleness check on the created entity is an attempt to satisfy the following constraints:

1. I would like sufficiently detailed fixtures to exercise any arbitrary part of the fetch code in multiple test cases without writing lots of edgy setup code
1. I would like to not recreate all of these fixtures every time I run a test case for a read-only fetch operation, of which there are a large and increasing number
2. I _would_ like to recreate those fixtures if I am running test suites that include only read-only tests, but have stopped and modified the fixture code in between test runs.

There may be a more elegant way to accomplish those last two concurrently, but just saying "If nobody has cleared the database in at least 30 seconds, it's probably time to rebuild" seemed like it did the job.